openpype/plugins/publish/extract_review_slate.py

```diff
@@ -82,12 +82,60 @@ def process(self, instance):
+            input_audio = False
+            audio_channels = None
+            audio_sample_rate = None
+            audio_channel_layout = None
             for stream in video_streams:
-                if "width" in stream and "height" in stream:
-                    input_width = int(stream["width"])
-                    input_height = int(stream["height"])
-                    break
-
+                self.log.debug("__ ffprobe: {}".format(stream))
+                if "codec_type" in stream:
+                    if stream["codec_type"] == "video":
+                        if stream["tags"]["timecode"]:
+                            # get timecode of the first frame
+                            input_timecode = stream["tags"]["timecode"]
+                            self.log.debug("__Video Timecode : {}".format(
+                                input_timecode))
+                        if "width" in stream and "height" in stream:
+                            input_width = int(stream["width"])
+                            input_height = int(stream["height"])
+                        if "r_frame_rate" in stream:
+                            # get frame rate in a form of
+                            # x/y, like 24000/1001 for 23.976
+                            input_frame_rate = str(stream["r_frame_rate"])
+                    if stream["codec_type"] == "audio":
+                        # get audio details
+                        # for generating silent audio track for slate
+                        if stream["channels"]:
+                            audio_channels = str(stream["channels"])
+                        if stream["sample_rate"]:
+                            audio_sample_rate = stream["sample_rate"]
+                        if stream["channel_layout"]:
+                            audio_channel_layout = stream["channel_layout"]
+            # calculate duration of one frame in seconds
+            if input_frame_rate:
+                # it is halved to make sure audio will be shorter then video
+                one_frame_duration = str(
+                    float(1.0 / eval(input_frame_rate)) / 2
+                )
+            else:
+                # same sane default (1 frame @ 25 fps)
+                one_frame_duration = 0.04
+            self.log.debug(
+                "One frame duration is {} sec".format(one_frame_duration))
+            # confirm we gathered all needed audio parameters
+            if audio_channel_layout:
+                if audio_sample_rate:
+                    if audio_channels:
+                        input_audio = True
+                        self.log.debug("__Audio : channels {}".format(
+                            audio_channels))
+                        self.log.debug("__Audio : sample_rate {}".format(
+                            audio_sample_rate))
+                        self.log.debug("__Audio : channel_layout {}".format(
+                            audio_channel_layout))
```

Comment:
Join conditions in one
```suggestion
            if (
                audio_channel_layout
                and audio_sample_rate
                and audio_channels
            ):
                input_audio = True
                self.log.debug("__Audio : channels {}".format(
                    audio_channels))
                self.log.debug("__Audio : sample_rate {}".format(
                    audio_sample_rate))
                self.log.debug("__Audio : channel_layout {}".format(
                    audio_channel_layout))
```

Reply:
Will do, thank you.